src/components/Sidebar/Sidebar.jsx

```diff
@@ -51,6 +57,49 @@ function Sidebar() {
           </button>
         </a>
       </div>
+      <div className="pt-5">
+        <button
+          className="inline-block cursor-pointer rounded-lg border-2 border-textSecondary bg-textSecondary px-[15px] py-1.5 text-center font-poppoins text-sm transition-all duration-500 hover:bg-transparent hover:text-textSecondary dark:text-white"
+          onClick={toggleContactForm}
+        >
+          Contact Us
+        </button>
+      </div>
+      {showContactForm && (
+        <div className="contact-container pt-5">
+          <h1 className="contact-heading">Contact Us</h1>
+          <p className="contact-description">Please fill out the form below to get in touch with us.</p>
+          <form className="contact-form">
+            <div className="form-group">
+              <label htmlFor="name" className="form-label">
+                Name:
+              </label>
+              <input type="text" id="name" name="name" className="form-input" />
+            </div>
+
+            <div className="form-group">
+              <label htmlFor="email" className="form-label">
+                Email:
+              </label>
+              <input type="email" id="email" name="email" className="form-input" />
+            </div>
+
+            <div className="form-group">
+              <label htmlFor="message" className="form-label">
+                Message:
+              </label>
+              <textarea id="message" name="message" rows="4" className="form-textarea" />
+            </div>
+
+            <button type="submit" className="submit-button" color="orange">
+              Submit
+            </button>
+          </form>
```

Comment:
How are you handling the form submissions?

Reply:
using Inline html css. Just make UI